app/src/main/res/values-zh-rCN/strings.xml

```diff
@@ -876,7 +876,7 @@
     <string name="sunevent_title">太阳</string>                   <!-- (custom) sun event title; e.g. "Sun 6°" -->
     <string name="sunevent_title_format">%1$s %2$s</string>      <!-- (custom) sun event title; e.g. "Sun 6°" -->
     <string name="sunevent_summary_format"><xliff:g id="title">%1$s</xliff:g> 于 <xliff:g id="angle">%2$s</xliff:g></string>    <!-- (custom) sun event summary; e.g. "Sun 6°" -->
-    <string name="sunevent_phrase_gender">其他</string>         <!-- either 'other' or 'female' (SelectFormat api) -->
```

Comment:
These `_gender` values are unusual. Translations can override them, but the expected value is always 'other' or 'female', which allows matching gender/plurality between some noun and its articles. 

For example, `offset_before_msg1` defines 
```{0} before {1, plural, one {{2, select, female {} other {}}} other {{2, select, female {} other {}}}}{3}```
 (which is basically empty / does nothing).

The Spanish version them overrides this with ```{0} antes {1, plural, one {{2, select, female {de la} other {del}}} other {{2, select, female {de las} other {de los}}}} {3}```. The provided `_gender` determines if this prints `de la <noun>` vs `del <noun>`, etc. 

https://developer.android.com/reference/android/icu/text/SelectFormat 
Its really confusing in my opinion, and since most languages don't seem to need them, I'll probably comment them out everywhere.